src/inmanta/env.py

```diff
@@ -101,7 +245,7 @@ def __init__(self, env_path: str) -> None:
         self._site_packages_dir: Optional[str] = None
 
     def get_package_installed_in_parent_env(self) -> Optional[Dict[str, str]]:
```

Comment:
is this dead code now?

Reply:
It looks like it, though not due to this pull request. Should I remove it?